tests/validation/tests/v3_api/test_custom_host_reg.py

```diff
@@ -73,8 +74,17 @@ def test_deploy_rancher_server():
 
     token = set_url_password_token(RANCHER_SERVER_URL,
                                    version=RANCHER_SERVER_VERSION)
-    admin_client = rancher.Client(url=RANCHER_SERVER_URL + "/v3",
-                                  token=token, verify=False)
+
+    retries = 5
```

Comment:
Can we add a sleep of a few seconds? Or just poll for a few seconds maybe 30 seconds max instead of a constant number 5 ?

Reply:
@sowmyav27 please review the changes. The loop with try getting admin_client for max 30 seconds.